clarity.d.ts

```diff
@@ -303,6 +304,14 @@ interface IClarityAssertFailedEventState extends IInstrumentationEventState {
 interface IClarityDuplicatedEventState extends IInstrumentationEventState {
   currentImpressionId: string;
 }
+
+interface IShadowDomInconsistentEventState extends IInstrumentationEventState {
```

Comment:
what are these value of the properties of these events? Can you add more comments about what this event is recording?